libs/elements/layout/images/src/lib/elewa-image-container/elewa-image-container.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="home-page">
+  <div *ngFor="let config of imageConfigs">
+    <h2 class="visual">
+      {{ getImageVisualisationClass(config.visualisation) }}
+    </h2>
```

Comment:
to add dynamic classes use ng-class directive or bind the class. interpolation doesn't work. If you were trying to just get the title then you could have achieved this directly